pages/getting-started/developers.md

```diff
@@ -211,7 +211,12 @@ NOTE: If you plan on upgrading to newer versions of the Standards in the future,
 
 ## Notes on accessibility
 
-We’ve designed the Standards to support older and newer browsers through progressive enhancement, and they officially support Internet Explorer 9 and up, along with the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage, and most agency websites should be able to safely begin support at Internet Explorer 9.
+- Originally the Standards were desiged to support Internet Explorer 9 and up
+- Now we're going to support any browser above 2% usage as defined on analytics.gov
```

Comment:
Period at the end of this sentence. Also, should we hyperlink analytics.gov to the website?  Or specifically hyperlink it to whatever part of analytics.gov shows the different browser percentages?

Reply:
I'm not sure there is a hyperlink to just that section.

Reply:
The URL should be analytics.usa.gov